.docker/subzero.Dockerfile

```diff
@@ -0,0 +1,60 @@
+# Note: We don't use Alpine and its packaged Rust/Cargo
```

Comment:
Already have this file at `docker/subzero.Dockerfile`. Probably should leave only one.

Reply:
docker folder is removed, still draft also, will ping again, have to change some things